extra/import_data.py

```diff
@@ -65,42 +69,45 @@ def __call__(self, parser, namespace, fname, option_string=None):
     # required arguments and optional arguments for the terminal. A help page is automatically generated.
     parser = DefaultHelpParser(
         description=(
-        "VUSualizer import data:\n"
-        "Imports data from 'O:\AMG_bijlagen_WES_uitslagen' after transferring this data into\n"
-        "the Dwergeik UMCU server. An additional script will notice new files in this folder\n"
-        "and use these as a parameter for this script. This will update the local MongoDB."
-        ),        
+            "VUSualizer import data:\n"
+            "Imports data from 'O:\AMG_bijlagen_WES_uitslagen' after transferring this data into\n"
+            "the Dwergeik UMCU server. An additional script will notice new files in this folder\n"
+            "and use these as a parameter for this script. This will update the local MongoDB."
+        ),
         formatter_class=argparse.RawTextHelpFormatter)
     parser.add_argument(
-        "-v", 
-        "--version", 
-        action='version', 
+        "-v",
+        "--version",
+        action='version',
         version='%(prog)s 1.0')
     # puts following arguments under 'required', rather than the default 'optional'
     required_argument = parser.add_argument_group('required arguments')
     # add other file-extensions if needed: for example: CheckExt({'xlsx', 'xls})
     required_argument.add_argument(
-        "-f", 
-        "--file", 
-        metavar='', 
-        required=True, 
-        nargs='+', 
+        "-f",
+        "--file",
+        metavar='',
+        required=True,
+        nargs='+',
         action=CheckExt({'xlsx'}),
-        help=".xlsx input, usage: -f /path/to/files/XXX.xlsx (or if multiple files, seperate with space: -f /path/to/files/XXX.xlsx /path/to/files/XXX2.xlsx'" )
+        help=".xlsx input, usage: -f /path/to/files/XXX.xlsx (or if multiple files, seperate with space: -f /path/to/files/XXX.xlsx /path/to/files/XXX2.xlsx'")
```

Comment:
needed an extra ), because of the function around it "DefaultHelpParser()"
Very nice suggestion. It will keep this line within the 127 line length rule